webpack.dev.js

```diff
@@ -6,8 +6,8 @@ const common = require("./webpack.common");
 
 module.exports = (env) => {
   return merge(common(env), {
-    mode: "development",
-    devtool: "eval-source-map",
+    mode: env.env === "production" ? "production" : "development",
+    devtool: env.env === "production" ? "source-map" : "eval-source-map",
```

Comment:
key change: production mode actually uses production mode.  TODO: see how easy it is to debug using source-map in production while still getting higher performance of production mode.